STYLE_GUIDE.md

```diff
@@ -223,39 +223,27 @@ A procedure is a series of numbered steps that a user follows to complete a spec
-
-Following is an example of procedure phrasing and formatting from Amazon EC2.
-
-![Procedure example](/images/procedures.PNG)
+#### Describing interactions with the UI
+
+We follow the _Microsoft Writing Style Guide_ guidance on describing interactions with a UI, provided here.
+
+Users interact with products using different input methods: keyboard, mouse, touch, voice, and more. So use generic verbs that work with any input method. Don't use input-specific verbs, such as _click_ or _swipe_. Instead, use the verbs listed here.
+
+| Verb | Use for | Examples |
+| :--------- | :------- | :------- |
+| **Open** | - Apps and programs <br> - Files and folders <br> - Shortcut menus <br> Use for websites and webpages only when necessary to match the UI. Otherwise, use _go to_. <br> - Don't use for commands and menus. | - Open Photos. <br> - Open the Reader app. <br> - Open the Filename file. <br> - To open the document in Outline view, select **View** > **Outline**. <br> - In WindowName, open the shortcut menu for ItemName. |
+| **Close** | - Apps and programs <br> - Dialog boxes <br> - Files and folders <br> - Notifications and alerts <br> - Tabs <br> - The action a program or app takes when it encounters a problem and can't continue. (Don't confuse with _stop responding_). | - Close the Alarms app. <br> - Close Excel. <br> - Save and close the document. <br> - Closing Excel also closes all open worksheets. |
+| **Leave** | Websites and webpages | Select **Submit** to complete the survey and leave this page. |
+| **Go to** | - Opening a menu. <br> - Going to a tab or another particular place in the UI. <br> - Going to a website or webpage. <br> - It's ok to use _On the **XXX** tab_ if the instruction is brief and continues immediately. | - Go to Search, enter the word **settings**, and then select **Settings**. <br> - Go to **File**, and then select **Close**. <br> - On the ribbon, go to the **Design** tab. <br> - Go to the **Deploy** tab. in the **Configuration** list ... <br> - On the **Deploy** tab, in the **Configuration** list ... <br> - Go to Example.com to register. |
+| **Select** | Instructing the user to select a specific item, including: <br> - Selecting an option, such as a button. <br> - Selecting a check box. <br> - Selecting a value from a list box. <br> - Selecting link text to go to a link. <br> - Selecting an item on a menu or shortcut menu. <br> - Selecting an item from a gallery. <br> - Selecting keys and keyboard shortcuts. (Document keyboard shortcuts only if they're the most likely way the customer will accomplish a task or as an alternative input method.) | - Select the **Modify** button. <br> - For **Alignment**, select **Left**. <br> - Select the text, open the shortcut menu, and then select **Font**. <br> - Select **Open in new tab**. <br> - Select the **LinkName** link. <br> - Select **F5**. <br> - Select **Shift+Enter**. <br> - Select **Ctrl+Alt+Delete**. |
```

Comment:
I personally prefer "press" when referring to keyboard shortcuts. Could we add it back?

Reply:
I added a row for "press" and moved the keyboard shortcut guidance/examples there. Can you confirm my change addresses your comment?

Reply:
Perfect, thank you!